package-lock.json

```diff
@@ -13,18 +13,18 @@
       }
     },
     "@babel/core": {
-      "version": "7.6.4",
-      "resolved": "https://registry.npmjs.org/@babel/core/-/core-7.6.4.tgz",
-      "integrity": "sha512-Rm0HGw101GY8FTzpWSyRbki/jzq+/PkNQJ+nSulrdY6gFGOsNseCqD6KHRYe2E+EdzuBdr2pxCp6s4Uk6eJ+XQ==",
+      "version": "7.7.2",
```

Comment:
Great comment and something I wondered about. I'm guessing I got both from using a combination of yarn install and npm install. I'll stick with just yarn since the netlify build just uses yarn install

Reply:
Yea, you'll get the package-lock file from 'npm install', so just add it to the git ignore files and delete it from your file system